core/scripts/chaincli/handler/keeper.go

```diff
@@ -596,3 +661,42 @@ func (k *Keeper) getConfigForRegistry12() *registry12.Config {
 		Registrar:            common.HexToAddress(k.cfg.Registrar),
 	}
 }
+
+func (k *Keeper) changeDirectory(directory string) (string, error) {
+	// Get the current working directory
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return "", fmt.Errorf("failed to get current working directory: %w", err)
+	}
+
+	// Calculate the absolute path of the target directory
+	absPath := filepath.Join(currentDir, directory)
+
+	// Check if already in the target directory
+	if currentDir == absPath {
+		// Check if hardhat.config.js exists in the current directory
+		_, err := os.Stat(filepath.Join(currentDir, "hardhat.config.js"))
```

Comment:
Should the file extension be .ts, so `hardhat.config.ts`?

Reply:
good catch!